cockroachcloud/security-overview.md

```diff
@@ -5,93 +5,158 @@ toc: true
+    <td>✓</td>
+    <td><a href="../{{site.versions['stable']}}/manage-certs-revoke-ocsp.html">OCSP</a> certificate revocation protocol</td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td><a href="../{{site.current_cloud_version}}/cluster-api.html">Cluster API</a> access using login tokens</td>
+  </tr>
+  <tr>
+  <td rowspan="5" >Data Protection</a></td>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Encryption-in-flight using TLS 1.3</td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Automatic backups for AWS clusters are encrypted-at-rest using <a href="https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingServerSideEncryption.html">AWS S3’s server-side encryption</a></td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Automatic backups for GCP clusters are encrypted-at-rest using <a href="https://cloud.google.com/storage/docs/encryption/default-keys">Google-managed server-side encryption keys</a></td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Industry-standard encryption-at-rest provided at the infrastructure level by your chosen deployment environment, such as Google Cloud Platform (GCP), Amazon Web Services (AWS), or Microsoft Azure.
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td><a href="cmek.html">Customer Managed Encryption Keys (CMEK)</a>.
+  </tr>
+  <tr>
+  <td rowspan="4" ><a href="authorization.html">Access Control (Authorization)</a></td>
+    <td>✓</td>
+    <td>✓</td>
+    <td>SQL users with direct privilege management</td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>SQL Role-based access control (RBAC)</td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Cloud Organization users with legacy roles</td>
+  </tr>
+  <tr>
+    <td>✓</td>
+    <td>✓</td>
+    <td>Cloud Organization users with fine-grained access roles</td>
+  </tr>
+  <tr>
+  <td rowspan="6">Network Security</td>
+    <td>✓</td>
+    <td>✓</td>
+    <td><a href="authentication.html">SQL-level configuration of allowed authentication attempts by IP address</a></td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td><a href="https://www.cockroachlabs.com/docs/cockroachcloud/private-clusters">Private Clusters</a></td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td>Network-level Configuration of allowed IP addresses</td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td>Egress Perimeter Controls</td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td><a href="network-authorization.html#vpc-peering">VPC Peering</a> for GCP clusters</td>
+  </tr>
+  <tr>
+    <td>&nbsp;</td>
+    <td>✓</td>
+    <td><a href="network-authorization.html#aws-privatelink"></a>PrivateLink for AWS clusters </td>
+  </tr>
+  <tr>
+    <td><a href="https://en.wikipedia.org/wiki/Non-repudiation">Non-Repudiation</a></td>
+    <td>✓</td>
+    <td>✓</td>
+    <td><a href="../{{site.versions['stable']}}/sql-audit-logging.html">SQL Audit Logging</a></td>
```

Comment:
There's also Org Audit Logs - https://www.cockroachlabs.com/docs/cockroachcloud/cloud-org-audit-logs.html, which apply to both.

Reply:
ok